### PR TITLE
Activate menu: standardize to 2 checkboxes

### PR DESCRIPTION
**Pull request by hius07**
3 states, 2 checkboxes

<kbd>MenuActivate</kbd>

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8017)

### CODE REVIEW

**Comment by poire-z** [comment]
(Hope you made sure we can't get with no checkbox checked, and can't get to the menu anymore :)

**Comment by hius07** [comment]
> Hope you made sure we can't get with no checkbox checked

Of course

File: frontend/ui/elements/menu_activate.lua [hunk 1]
```diff
@@ -2,55 +2,36 @@ local InfoMessage = require("ui/widget/infomessage")
...
-                    return true
-                else
-                    return false
-                end
+                return G_reader_settings:readSetting("activate_menu") ~= "swipe"
             end,
             callback = function()
-                G_reader_settings:saveSetting("activate_menu", "swipe_tap")
-                UIManager:show(InfoMessage:new{
-                    text = _("This will take effect on next restart."),
```
Comment (Frenzie) [inline frontend/ui/elements/menu_activate.lua:23]: Changes don't require a restart?

Comment (hius07) [reply frontend/ui/elements/menu_activate.lua:23]: There were 3 checkboxes, now remain only 2, so one InfoMessage is removed.

**Pull request merged**